asa/ja-jp/Content/Topics/ReleaseNotes/advanced-server-access-release-notes.htm

```diff
@@ -328,6 +328,8 @@ <h1 class="relnotes">未翻訳のドキュメント</h1>
                                     </div>
                                     <!-- ********************** Scripts ************************************* -->
                                     <!-- Foundation -->
+                                    <script src="../../Resources/Scripts/js/ignore-dompurify.js">

```

Comment:
https://oktainc.atlassian.net/browse/OKTA-538286 should take care of automating this (if needed)

Reply:
In this case, I believe we added the `ignore-dompurify.js` script to the EN doc after we sent this to translation. So, it'll be automatically included in future rounds.